.changeset/strange-years-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/ag-grid-theme": patch
+---
+
+Update header icon alignment
```

Comment:
Be more specific about the change? around left margin -> center align?

Reply:
How does this sound?